### PR TITLE
Implement server-side logic for comments page

### PR DESCRIPTION
**Pull request by lin8231**
* Form submissions are sent to Datastore
* Comments list is retrieved from Datastore
* Empty comment entries are disallowed (20 char min)
* Empty name is allowed - assigned default value "Anonymous"

* Delete all comments functionality is not and will not be implemented (see Week 3 Project Planning)

* Server playground files are deleted

### CODE REVIEW

File: portfolio/src/main/webapp/style/commentsPage.css [hunk 1]
```diff
@@ -22,10 +22,67 @@
...
+
+.formLabel {
+    /* align-self: flex-start; */
+    font-family: 'Raleway', sans-serif;
+    font-size: 1.5vw;
+    font-weight: 300;
+}
+
+.textField {
+    /* width: 100%; */
```
Comment (leonnemets) [inline portfolio/src/main/webapp/style/commentsPage.css]: remove comments

File: portfolio/src/main/java/com/google/sps/Comment.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+public final class Comment {
```
Comment (leonnemets) [inline portfolio/src/main/java/com/google/sps/Comment.java:3]: Consider using autovalue to generate this class https://github.com/google/auto/blob/master/value/userguide/index.md

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 3]
```diff
@@ -20,13 +19,13 @@
...
 @WebServlet("/listComments")
-public class listCommentsServlet extends HttpServlet {
+public class ListCommentsServlet extends HttpServlet {
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort("time", SortDirection.DESCENDING);
 
-    DatastoreService datastore = new DatastoreServiceFactory.getDatastoreService();
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```
Comment (leonnemets) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:28]: nice

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 4]
```diff
@@ -28,8 +28,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
+    int maxNum = Integer.parseInt(request.getParameter("numComments"));
```
Comment (leonnemets) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:31]: Can this be added to the query itself? you potentially fetching lots from the database only to send back a few

**Review by leonnemets — APPROVED** [review]

File: portfolio/src/main/webapp/style/commentsPage.css (hunk 1 above)
Comment (lin8231) [reply portfolio/src/main/webapp/style/commentsPage.css]: Removed in a later commit :)

File: portfolio/src/main/java/com/google/sps/Comment.java (hunk 2 above)
Comment (lin8231) [reply portfolio/src/main/java/com/google/sps/Comment.java:3]: Thanks for the suggestion! Will look into this for the next PR

**Pull request merged**